app/routes/app/brainstorm/$id.challenges.tsx

```diff
@@ -71,10 +70,10 @@ export default function MarketplaceChallenges() {
             </div>
           </Detail>
         </div> */}
-          <Text color="dimmed" className="max-w-2xl">
+          <div className="max-w-2xl text-[#666666] text-[14px]">
```

Comment:
Can just do `text-sm` instead of doing a custom size here. Also we're avoiding custom color hexes where possible and using the [Tailwind palettes](https://tailwindcss.com/docs/customizing-colors). I think for these we've been using `text-gray-500`.

Oh and it might also be more semantic to use a `p` tag here instead of `div`.

Reply:
oops good catches. Nice. Will continue this pattern of the tailwind palettes etc